integreat_cms/cms/templates/pois/poi_form_sidebar/related_contacts_box.html

```diff
@@ -17,12 +17,38 @@
                     {% trans "This location is not currently referred to in any contact." %}
                 {% endif %}
             </div>
-            {% for contact in contacts %}
-                <a href="{% url 'edit_contact' contact_id=contact.id region_slug=request.region.slug %}"
-                   class="block pt-2 hover:underline">
-                    <i icon-name="pen-square" class="mr-2"></i> {{ contact.label_in_reference_list }}
-                </a>
-            {% endfor %}
+            <div id="related-contact-list">
+                {% for contact in contacts %}
+                    <a href="{% url 'edit_contact' contact_id=contact.id region_slug=request.region.slug %}"
+                       class="block pt-2 hover:underline">
+                        <i icon-name="pen-square" class="mr-2"></i> {{ contact.label_in_reference_list }}
```

Comment:
```suggestion
                        <i icon-name="pencil" class="mr-2"></i> {{ contact.label_in_reference_list }}
```
At least for the list views we changed the icon to the pencil, but maybe it's okay for this view because the context is slightly different? (And also: we still use this icon still in a few places)